pages/facts.html

```diff
@@ -1,12 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <link href="../styles/facts.css" rel="stylesheet">
+    <link href="../styles/style.css" rel="stylesheet">
+    <title>Benefits of Having a Pet</title>
+  </head>
+  <body>
+    <nav>
+      <a href="index.html">Home</a>
+      <a href="gallery.html">Gallery</a>
+    </nav>
+    <section class="grid-container">
+      <h1>Benefits of Having a Pet</h1>
+      <ul>
+        <li>Emotional Support: Pets can provide emotional support and companionship, reducing feelings of loneliness and depression.</li>
+        <li>Physical Health: Regular interaction with a pet can improve physical health by reducing blood pressure, improving cardiovascular health, and boosting immunity.</li>
+        <li>Mental Health: Caring for a pet can help to reduce stress and anxiety and improve overall mental health.</li>
+        <li>Socialization: Owning a pet can encourage socialization and increase opportunities for social interactions with other pet owners.</li>
+        <li>Security: Some pets, such as dogs, can provide a sense of security and protection for their owners.</li>
+        <li>Unconditional Love: Pets can provide unconditional love and companionship, offering a source of comfort and joy in times of need.</li>
+      </ul>
+    </section>
+      <figure class="flex-item">
+      <!-- <img src="https://pbs.twimg.com/media/FFi_ax7XsA0fvYf?format=jpg&name=large" width="600" 
+        height="500" alt="cat doing code on laptop"> -->
+      <img src="https://preview.redd.it/0kxkc7luqbc61.jpg?auto=webp&s=771ae62f18c007e3919eec2d100a6d66891c0447" width="500" 
```

Comment:
Typically, we would try to keep the width and the height to the css instead of in the html!